modules/workbench/modules/egress_detection/assets/content/egress_window_template.json

```diff
@@ -0,0 +1,101 @@
+{
+  "type": "SavedSearchWithScheduleSyncDefinition",
+  "name": "${aou_env} / ${tier_name}-tier / ${egress_threshold_mib}-Mib / ${egress_window_sec}-min egress event",
+  "search": {
+    "queryText": ${query_text},
+    "defaultTimeRange": "-120m",
+    "byReceiptTime": true,
+    "viewName": "",
+    "viewStartTime": "1970-01-01T00:00:00Z",
+    "queryParameters": [
+      {
+        "name": "environment",
+        "label": "environment",
+        "description": "",
+        "dataType": "QUERY_FRAGMENT",
+        "value": "${aou_env}",
+        "autoComplete": {
+          "autoCompleteType": "SKIP_AUTOCOMPLETE",
+          "autoCompleteKey": null,
```

Comment:
Do we need these parameters? Or does it generate diffs without?

Reply:
The whole json file and query text was downloaded form sumoLogic UI. I feel it is safer to just keep it as it?

Reply:
Sounds fine. Hopefully we don't generate diffs when the API specification changes slightly (adds a new field)